src/hotspot/share/opto/runtime.hpp

```diff
@@ -242,95 +308,365 @@ class OptoRuntime : public AllStatic {
   // Type functions
   // ======================================================
 
-  static const TypeFunc* new_instance_Type(); // object allocation (slow case)
-  static const TypeFunc* new_array_Type ();   // [a]newarray (slow case)
-  static const TypeFunc* new_array_nozero_Type ();   // [a]newarray (slow case)
+  static inline const TypeFunc *new_instance_Type() {
```

Comment:
Same: please, spell it as `TypeFunc*` (and subsequent declarations).

Reply:
updated.